src/components/MultiSuggestField/tests/constants.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback, useState } from "react";
+import { loremIpsum } from "react-lorem-ipsum";
+
+import { MultiSelectSelectionProps } from "../../MultiSelect/MultiSelect";
```

Comment:
Please always import from our root index. This way we test if our elements and interfaces are exposed correctly through the package main export scripts.